### ISSUE TITLE
[UX] Add a more friendly alternative for the Address_index of the derivation path

### ISSUE THREAD

**Issue by albertov19** [polkadot-js/apps#4736]
When users use a mnemonic seed with ECDSA the default derivation path is `m/44'/60'/0'/0/0`. The user needs to manually increment the address_index to get the subsequent address from the same mnemonic seed.

It would a good idea to implement either a dropdown menu that shows all the addresses (either by groups of 5) or just use the next one (if it finds that the previous index has been used).

This is similar to the behavior when using MetaMask, from a mnemonic seed you just click on a button that says "Add Account" and automatically creates an address from the next address_index of the derivation path.

In such a way users will be able to just add multiple accounts from the same mnemonic seed without needing to know/understand how derivation paths work.

**Comment by jacogr** [comment]
I am assuming you are referring to the Frontier-like chains here which are using Ethereum-compatible keys, e.g. Moonbeam? The standard Substrate Ecdsa doesn't use the bip39 derivation paths.

Willing to accept PRs for it, e.g. in the Ledger derivation this is exactly what is done - it "hides" the full path until right at the end.

**Comment by albertov19** [comment]
Yes you are totally right, I was referring to Ethereum-compatible keys. I'm with the Moonbeam team :) Thanks Jaco!

**Comment by joelamouche** [comment]
>In such a way users will be able to just add multiple accounts from the same mnemonic seed without needing to know/understand how derivation paths work.

But just to be clear: the mnemonic is randomly generated everytime, so one would have to re-enter the mnemonic to derive multiple accounts from the same mnemonic

**Comment by jacogr** [comment]
I would suggest connecting to Polkadot/Kusama and try the Ledger paths - they are not perfect, but may give some ideas. (At least wopuld align it with MM-like selection, so would lean towards the defaults they have)

**Comment by albertov19** [comment]
@joelamouche Yes but let's say I have my mnemonic from MetaMask for example, or I have it written down or whatever and I want to import multiple accounts from it.

**Comment by joelamouche** [comment]
@jacogr how do I try the ledger path? Where is it on the app?

**Comment by jacogr** [comment]
Connect to one of the Ledger supported chains - Kusama, Polkadot. Dock and add an account. Then select the type as Ledger-compatible.

**Comment by joelamouche** [comment]
>Then select the crypto type as Ledger-compatible.

How do I do that? I wasn't presented with this option when I added an address

**Comment by jacogr** [comment]
Think it may not show unless you have selected to activate Ledger under settings. Here it is, simpler -

![image](https://user-images.githubusercontent.com/1424473/109522204-52623c80-7aae-11eb-8e82-f7cdf5a68d15.png)

**Comment by jwierzbo** [comment]
I do not see "Ledger" option anymore at https://polkadot.js.org/apps/
How to import PolkaDOT from ledger using mnemonics now?

**Comment by joelamouche** [comment]
I'm not sure what you are trying to do: you can use "add account" to add an account using a mnemonic

**Comment by jwierzbo** [comment]
I'm trying to import my Ledger PolkaDOT account which is using `m/44'/354'/account'/0'/0'` derivation path to the polkadot.js app.
i'm looking how to define properly derivation path in polkadot.js to get the same address as in Ledger

**Comment by joelamouche** [comment]
Oh I see 👍

**Comment by albertov19** [comment]
You can select your custom path by selecting custom derivation path. However, MetaMask uses m/44'/60'/0'/0/0 so if you use a custom derivation path you'll obtain a different address than if you then import your account to metamask

![image](https://user-images.githubusercontent.com/64150856/135998582-fe4bad3b-6532-4adf-b2e0-83e3b0d395c0.png)

![image](https://user-images.githubusercontent.com/64150856/135998734-b5e0b7a6-e78c-4fb7-a63d-6560bd357a29.png)

**Comment by jwierzbo** [comment]
This Add windows looks different on https://polkadot.js.org/apps <https://polkadot.js.org/apps>

**Comment by albertov19** [comment]
@jwierzbo yes I deleted my comment because that was the `addAccount` window from Moonriver/Moonbeam, not Polkadot/Kusama. Sorry for the confusion
